server/config/express.js

```diff
@@ -146,7 +167,7 @@ module.exports = (app, config) => {
 		app.use((err, req, res, next) => {
 			const status = err.status || (isCelebrate(err) ? 400 : 500);
 			if (status === 500 || isCelebrate(err))
-				console.error(err);
```

Comment:
For the test environment, we should probably have this still be console. Tests are only run locally or in CI, not on production.